### PR TITLE
feat(initialize): Add skeleton files

### PR DESCRIPTION
**Pull request by mamoonraja**
What's Left:

- [x] Add continous integrations
- [x] Add method for kube
- [x] Add method for Starter kit
- [x] Add method for local config file
- [x] Tests
- [x] add a linter 
- [x] get 100% code coverage
- [x] Support all the features from the node.js package
- [x] Add semantice release
- [x] Update Readme & documentation

Coverage:
screen shot 2018-07-31 at 6 00 04 pm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.4"
```
Comment (germanattanasio) [inline .travis.yml]: 2.7 should be added

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+setup(name='vcap_services',
+      version=__version__,
+      description='Client library to use VCAP SERVICES',
+      license='Apache 2.0',
+      author='Mamoon Raja',
+      author_email='mamoon.raja@.ibm.com',
+      long_description=read_md('README.md'),
+      url='https://github.com/mamoonraja/vcap-services-python',
+      packages=['vcap_services'],
```
Comment (germanattanasio) [inline setup.py]: `vcap-services`not `vcap_services`

File: test/test_vcap-services.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from vcap_services import parse_credentials, load_from_vcap_services, get_creds_for_starter
```
Comment (germanattanasio) [inline test/test_vcap-services.py]: `test/test_parse_credentials.py`

File: vcap_services/index.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    key = "watson_" + service_label + "_"
+    creds = {}
+    if credentials is not None:
+        cred_keys = credentials.keys()
+        filtered_keys = list(filter(lambda x: x.find(key) != -1, cred_keys))
+        for k in filtered_keys:
+            creds[k.replace(key, '')] = credentials[k]
+    return creds
+
+def get_creds_for_starter(service_label, credentials_from_local_config=None):
```
Comment (germanattanasio) [inline vcap_services/index.py]: `get credentials from file`

File: vcap_services/index.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    vcap_services = os.getenv("VCAP_SERVICES")
+    if vcap_services is not None:
+        services = json.loads(vcap_services)
+        if service_name in services:
+            return services[service_name][0]["credentials"]
+    else:
+        return None
+
+def parse_credentials(service_label, credentials):
+    key = "watson_" + service_label + "_"
```
Comment (germanattanasio) [inline vcap_services/index.py]: do we use single quote or double quote?

**Review by germanattanasio — CHANGES_REQUESTED** [review]
This is great progress.
Todo
- [x] add a linter 
- [x] get 100% code coverage
- [x] Support all the features from the node.js package

**Comment by germanattanasio** [comment]
You need to write tests for when things are None and to separate them in different tests rather than listing them all in one

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    os.system('python setup.py sdist upload -r pypitest')
+
+    # production server
+    os.system('python setup.py register -r pypi')
+    os.system('python setup.py sdist upload -r pypi')
+    sys.exit()
+
+# Convert README.md to README.rst for pypi
+try:
+    from pypandoc import convert
```
Comment (germanattanasio) [inline setup.py:36]: you are not installing `pypandoc`.

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+    def run_tests(self):
+        import pytest
+        errcode = pytest.main(self.test_args)
+        sys.exit(errcode)
+
+
+setup(name='vcap_services',
+      version=__version__,
+      description='Client library to use VCAP SERVICES',
```
Comment (germanattanasio) [inline setup.py]: Find a better description

**Review by germanattanasio — CHANGES_REQUESTED** [review]

File: setup.py (hunk 6 above)
Comment (mamoonraja) [reply setup.py:36]: added to `requirements.txt`

**Review by germanattanasio — APPROVED** [review]
